new_client/src/components/auth/Signin.jsx

```diff
@@ -69,61 +122,89 @@ function Signin(props) {
 			history.push("/");
 		}
 	}, [error, isAuthenticated, history]);
-
-	return (
-		<Container maxWidth="sm">
-			<h1 className="heading"> LOGIN </h1>
-			<Paper style={{ padding: 16 }} id="from_style">
-				<Grid container alignItems="flex-start" spacing={2}>
-					<Grid item xs={12}>
-						<TextField
-							fullWidth
-							required
-							name="username"
-							type="text"
-							placeholder="Username"
-							onChange={handleUsername}
-						/>
-						<div className="errorMsg">{errors.username}</div>
-					</Grid>
-
-					<Grid item xs={12}>
-						<TextField
-							fullWidth
-							required
-							name="password"
-							type="password"
-							placeholder="Password"
-							onChange={handlePassword}
-						/>
-						<div className="errorMsg">{errors.password}</div>
-					</Grid>
-
-					<Grid item xs={12}>
-						<Button
-							variant="contained"
-							color="secondary"
-							onClick={handleClick}
-						>
-							Submit
-                </Button>
-					</Grid>
-
-					<Grid item xs={12}>
-						<Typography>
-							New User ? <Link to="/signup">Sign Up</Link> instead.
-						</Typography>
-					</Grid>
-				</Grid>
-			</Paper>
-		</Container>
-
-	)
+  const classes = useStyles();
+  return (
+    <ThemeProvider theme={theme1}>
+      <Grid container style={{height:'87vh'}}>   
+      <Grid item sm={false} md={7} className={classes.image} />    
+        <Grid item sm={12} md={5}>
+    <Container component="main" maxWidth="xs">
+      <Paper className={classes.paper} elevation={3}>
+      <img style={{marginTop:20,height:'90%',width:'90%'}} src={coc}/>
+
+        <Typography style={{color:'#fff'}} component="h1" variant="h5">
+          Sign in
+        </Typography>
+        <form className={classes.form} noValidate>
+          <div className={classes.formInner}>
+          <TextField
+            margin="normal"
+            fullWidth
+            color='#52b107'
+            required
+            label="Username"
+            name="username"
+            style={{color:'#52b107',borderColor:'#52b107'}}
+            onChange={handleUsername}
+      autoFocus
+      InputProps = {{startAdornment: <InputAdornment position="start"><AccountCircle/></InputAdornment>}}
+          />
```

Comment:
We are missing a div here. If there are errors, they need to be shown by a `<div>`